### PR TITLE
Add provedState field to graphQL account DEVELOP

### PR DESCRIPTION
**Pull request by mitschabaude**
develop version of https://github.com/MinaProtocol/mina/pull/12172

### CODE REVIEW

**Review by bkase — CHANGES_REQUESTED** [review]
This should be a merge of the other branch rather than a reapply of the same PR

File: graphql_schema.json [hunk 1]
```diff
@@ -11658,6 +11658,15 @@
               "isDeprecated": false,
               "deprecationReason": null
             },
+            {
+              "name": "provedState",
+              "description":
+                "Boolean indicating whether all 8 fields on zkAppState where last set by a proof-authorized account update",
```
Comment (Trivo25) [inline graphql_schema.json:11664]: I believe `where` should be `were`

File: src/lib/mina_graphql/mina_graphql.ml [hunk 2]
```diff
@@ -1396,6 +1396,15 @@ module Types = struct
                    account.Account.Poly.zkapp
                    |> Option.map ~f:(fun zkapp_account ->
                           zkapp_account.app_state |> Zkapp_state.V.to_list ) )
+             ; field "provedState" ~typ:bool
+                 ~doc:
+                   "Boolean indicating whether all 8 fields on zkAppState \
```
Comment (Trivo25) [inline src/lib/mina_graphql/mina_graphql.ml:1401]: same here

**Review by Trivo25 — APPROVED** [review]

**Comment by mitschabaude** [comment]
closing since this should be brought to develop as part of a rampup -> develop merge

**Pull request closed without merging**